features/steps/partcad-cli/commands/version.py

```
@@ -5,46 +5,105 @@
+    substring = expandvars(substring, context)
+    logging.debug(f"STDERR: {context.result.stderr}")
+    logging.debug(f"STDOUT: {context.result.stdout}")
+    # TODO-74: @alexanderilyin: strip ASCII color codes from the output
+    if normalize_path(substring) in normalize_path(strip_ansi(context.result.stdout)):
+        raise AssertionError(
+            f"Expected '{normalize_path(substring)}' not to be in '{normalize_path(strip_ansi(context.result.stdout))}', but it was found"
+        )
 
 
 @then('STDOUT should not contain "{substring}"')
 def step_impl(context, substring):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 66

(F811)